server/src/main/java/server/haengdong/application/BillActionService.java

```diff
@@ -86,6 +102,7 @@ public void deleteBillAction(String token, Long actionId) {
         Event event = eventRepository.findByToken(token)
                 .orElseThrow(() -> new HaengdongException(HaengdongErrorCode.EVENT_NOT_FOUND));
 
+        billActionDetailRepository.deleteByBillAction_Action_EventAndBillAction_ActionId(event, actionId);
```

Comment:
```suggestion
        billActionDetailRepository.deleteAllByBillAction_Action_EventAndBillAction_ActionId(event, actionId);
```
r: BillActionDetail 여러 개 삭제하는 거니까 deleteAll을 사용하는 게 더 예측하기 쉬울 것 같아요.